python/ray/serve/controller.py

```diff
@@ -376,33 +409,31 @@ async def run_control_loop(self) -> None:
 
     def get_backend_configs(self) -> Dict[str, BackendConfig]:
         """Fetched by the router on startup."""
-        backend_configs = {}
-        for backend, info in self.backends.items():
-            backend_configs[backend] = info.backend_config
-        return backend_configs
+        return self.metadata_info.get_backend_configs()
 
     def get_traffic_policies(self) -> Dict[str, TrafficPolicy]:
         """Fetched by the router on startup."""
-        return self.traffic_policies
+        return self.metadata_info.traffic_policies
 
-    def _list_replicas(self, backend_tag: str) -> List[str]:
+    def _list_replicas(self, backend_tag: BackendTag) -> List[str]:
         """Used only for testing."""
-        return self.replicas[backend_tag]
+        return self.lifetime_info.replicas[backend_tag]
 
     def get_traffic_policy(self, endpoint: str) -> TrafficPolicy:
         """Fetched by serve handles."""
-        return self.traffic_policies[endpoint]
+        return self.metadata_info.traffic_policies[endpoint]
 
-    async def _start_backend_worker(self, backend_tag: str, replica_tag: str,
+    async def _start_backend_worker(self, backend_tag: BackendTag,
```

Comment:
can we move this to a member method in LifeTimeInfo? same for all the methods that start/stop/scale actors